prov/bgq/Makefile.include

```diff
@@ -0,0 +1,78 @@
+if HAVE_BGQ
+AM_CFLAGS += -Wno-ignored-qualifiers
+AM_CPPFLAGS += -I$(builddir)/prov/bgq/include -I$(srcdir)/prov/bgq/include
+_bgq_files = prov/bgq/src/fi_bgq_atomic.c \
+		prov/bgq/src/fi_bgq_av.c \
+		prov/bgq/src/fi_bgq_spi.c \
+		prov/bgq/src/fi_bgq_cm.c \
+		prov/bgq/src/fi_bgq_cntr.c \
+		prov/bgq/src/fi_bgq_cq.c \
+		prov/bgq/src/fi_bgq_domain.c \
+		prov/bgq/src/fi_bgq_ep.c \
+		prov/bgq/src/fi_bgq_fabric.c \
+		prov/bgq/src/fi_bgq_info.c \
+		prov/bgq/src/fi_bgq_init.c \
+		prov/bgq/src/fi_bgq_mr.c \
+		prov/bgq/src/fi_bgq_msg.c \
+		prov/bgq/src/fi_bgq_rma.c \
+		prov/bgq/src/fi_bgq_sep.c \
+		prov/bgq/src/fi_bgq_tagged.c \
+		prov/bgq/src/fi_bgq_node.c \
+		prov/bgq/src/fi_bgq_progress.c \
+		prov/bgq/src/fi_bgq_pmi.c \
+		prov/bgq/external/memory_impl.c
+bgq_CPPFLAGS += -I@bgq_driver@ -I@bgq_driver@/spi/include/kernel/cnk
+bgq_LDFLAGS += -static
+
+prov/bgq/external/memory_impl.c: @bgq_external_source@/spi/src/kernel/cnk/memory_impl.c
+	mkdir -p $(@D)
+	cp $< $@
```

Comment:
The size of the file is not the issue here.  GNU Autotools guidelines are that if a file is not included in the tarball, it should be cleaned as part of `make distclean`.